app/models/attachment.rb

```diff
@@ -2,9 +2,10 @@ class Attachment < ActiveRecord::Base
   belongs_to :user
   belongs_to :comment, counter_cache: true
 
-  validates_presence_of :filename, :location, :user_id
+  has_attached_file :file
+  do_not_validate_attachment_file_type :file
```

Comment:
This will need to change, but we'll want to figure out what files we'd like to support (I reckon a lot of them. Maybe everything but exe or something)